### PR TITLE
Script for calculating area and volume of some figures

### PR DESCRIPTION
**Pull request by fioalvsera**
Script that calculates the area of a square, a rectangle and a circle, and the volume of a cylinder, using functions

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,33 +2,81 @@
...
+    return base * height
+}
+
+function getCircleArea(radio){
+    return (radio * radio) * 3.14
+}
+
+var area = 0
+
+switch(shape){
```
Comment (jseravalli) [inline index.js]: Is there any reason to have 2 switch statements? I think you can merge them and have only one.

File: index.js [hunk 2]
```diff
@@ -2,33 +2,81 @@
 //node index.js square [side]
 
 var shape = process.argv[2] // [shape] cylynder || square
+var infoBase = process.argv[3]
+var infoHeight = process.argv[4]
 
-let area = 0
+function getCylinderVolume(radio, height){
+    let area = (radio * radio) * 3.14
```
Comment (jseravalli) [inline index.js]: Can you reuse the function to calculate the area of a circle here?

**Review by jseravalli — CHANGES_REQUESTED** [review]
I added two small requests but overall great job!! 🙌🙌🙌

File: index.js (hunk 1 above)
Comment (fioalvsera) [reply index.js]: I'll try to do it

File: index.js (hunk 2 above)
Comment (fioalvsera) [reply index.js]: I'll try to do it

File: index.js (hunk 1 above)
Comment (fioalvsera) [reply index.js]: I did it, is it ok?

File: index.js (hunk 2 above)
Comment (fioalvsera) [reply index.js]: I did it, it took me more time but I think I did it. Is it ok? Thank you

**Review by jseravalli — APPROVED** [review]

**Pull request merged**